app/views/chargeback_assignment/_form.html.haml

```diff
@@ -12,7 +12,7 @@
     .col-md-8
       - options = TermOfServiceHelper::ASSIGN_TOS[@edit[:new][:type] == "Compute" ? "chargeback_compute".to_sym : "chargeback_storage".to_sym].map do |k, v|
         - [_(v), k]
-      = select_tag("cbshow_typ", options_for_select([[nothing, "nil"]] + options, @edit[:new][:cbshow_typ]),
+      = select_tag("cbshow_typ", options_for_select([[nothing, "nil"]] + options, @edit[:new][:cbshow_typ]), "aria-label" => "Select term of service type",
```

Comment:
@amalvijayan03 , Could you mark the strings like this `_("Select term of service type")`

Reply:
Made the changes